website/docs/r/ldap.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: "turbot"
+title: "turbot"
+template: Documentation
+page_title: "Turbot: turbot_ldap_directory"
+nav:
+  title: turbot_ldap_directory
+---
+
+# turbot\_ldap\_directory
+
+The `Turbot LDAP Directory` resource adds support for ldap directories. It is used to create, manage and delete directory settings.
+
+## Example Usage
+
+###Creating Your First LDAP Directory
+
+```hcl
+resource "turbot_ldap_directory" "test" {
+  parent                = "tmod:@turbot/turbot#/"
+  title                 = "ldap Directory"
+  description           = "My first ldap directory"
+  profile_id_template   = "{{profile.email}}"
+  base                  = "test"
+  distinguished_name    = "ldap directory"
+  password              = ""
+  url                   = "test.com"
+ tls_enabled            = false
+ reject_unauthorized    = false
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `parent` - (Required) ID or `aka` of the parent resource.
+- `profile_id_template` - (Required) A template to generate profile id for users authenticated through a ldap directory. For example, email id of the user.
+- `password` - (Required) The password of the user specified in directory server.
+- `title` - (Required) Short and descriptive name to help identify the directory server.
+- `url` - (Required) The FQDN (fully qualified domain name) where the directory server is available. The FQDN also includes the protocol to be used for accessing the server.
+- `base` - (Required) The BaseDN of all the requests that are sent to LDAP server. In order for your users and groups to be found in an application, they must be located underneath the base DN.
+- `distinguished_name` - (Required) This is the username that Turbot will use to authenticate with the directory server after connection has been established. Mostly referred to BindDN in LDAP.
+- `tls_enabled` - (Required) TLS setting for the directory server makes sure which certificates should be accepted, rejects those are not signed by a trusted CA.(Boolean)
```

Comment:
is is standard practice to specify boolean properties like this?

Reply:
yes... referred to aws